activerecord/lib/active_record/migration.rb

```diff
@@ -688,24 +688,6 @@ def nearest_delegate # :nodoc:
         delegate || superclass.nearest_delegate
       end
 
-      # Raises ActiveRecord::PendingMigrationError error if any migrations are pending
-      # for all database configurations in an environment.
-      def check_all_pending!
-        pending_migrations = []
-
-        ActiveRecord::Tasks::DatabaseTasks.with_temporary_pool_for_each(env: env) do |pool|
-          if pending = pool.migration_context.open.pending_migrations
-            pending_migrations << pending
-          end
-        end
-
-        migrations = pending_migrations.flatten
-
-        if migrations.any?
-          raise ActiveRecord::PendingMigrationError.new(pending_migrations: migrations)
-        end
-      end
```

Comment:
I'm not sure why I didn't fix this when I opened that PR if this fix is correct. What is calling `check_all_pending!` in your app? It's not called from Rails.